assets/src/edit-story/app/media/utils/useTranscodeVideo.js

```diff
@@ -24,13 +24,17 @@ import { useFeature } from 'flagged';
  * Internal dependencies
  */
 import { useConfig } from '../../config';
+import { useUser } from '../../user';
 
 const isDevelopment = process.env.NODE_ENV === 'development';
 
 const getFileName = ({ name }) => name.split('.').slice(0, -1).join('.');
 
 function useTranscodeVideo() {
   const { ffmpegCoreUrl } = useConfig();
+  const {
+    actions: { getCurrentUser },
+  } = useUser();
```

Comment:
With the other suggestion this would simply become:

```js
  const { state: currentUser } = useUser();
```

Reply:
What if currentUser isn't loaded yet?

Reply:
Then `currentUser.meta?.web_stories_media_optimization` will return `false`

Reply:
> Then currentUser.meta?.web_stories_media_optimization will return false 

Which is a problem, as that is not the users preference.

Reply:
The user data is preloaded and the hook will use the updated value whenever it changes. So not really a problem.

Reply:
Updated.